### ISSUE TITLE
[NPC] Raven's Wood Leafbeard - interferes with quest 10830 (low prio issue imo)

### ISSUE THREAD

**Issue by mhdemn** [Atlantiss/NetherwingBugtracker#2522]
TL:DR There's a way to turn Raven's Wood Leafbeards friendly, also there's a quest that asks you to do this. They keep you in combat until they return to normal soon after. It's just a minor annoyance since other friendly-hostile NPCs for several quests out there bug you in combat for way longer, forcing you to alt+f4.

**Description**: In this area in BEM called Raven's Wood there's a bunch of Raven's Wood Leafbeards. If you have an [Exorcism Feather](https://db.excalibur.ws/?item=31518) (Which is created with a [Grishnath Orb](https://db.excalibur.ws/?item=31495) and a [Dire Pinfeather](https://db.excalibur.ws/?item=31517)) you can use it on them to summon the Koi-Koi Spirit that "haunts" them, so you can turn the Leafbeard into a friendly NPC. You can do all this stuff without having the quest that asks you to kill 5 of these spirits.

**Current behaviour**: Leafbeards will [keep you in combat](https://youtu.be/VeDB7QR1C2Y) until they return to hostile status 20-30 sec after they turned friendly (as a result of the Koi-Koi Spirit's death). They do not have you targeted when become hostile again; you exit combat mode a couple seconds after they returned to starting position.

**Expected behaviour**: Friendly Leafbeards do not interfere with your combat status in any way.

**Server Revision**: 2272

Related issue: #2436 - Is fixed now, see video above.

**Comment by Evolvee** [comment]
its a core issue, the workaround is to make the NPC evade @Wolffenstein but this affects way more events where combat NPC enters friendly state during the fight

**Comment by Evolvee** [comment]
@mhdemn the neutral thingy is intended --> once u reward the quest, they act neutral to you instead of hostile (its RP thing...)

**Comment by mhdemn** [comment]
> @mhdemn the neutral thingy is intended --> once u reward the quest, they act neutral to you instead of hostile (its RP thing...)

gotcha, makes sense. Thought it was an unintended change 'cause 2273 happened just before I turned in lol

**Comment by Godmatik** [comment]
Rev 4017 - still an issue
You can also summon multiple spirits from one tree. Doesn't give you extra quest credit, but should not be possible.
